### ISSUE TITLE
Sample bot-sso-adaptivecard still using botframeworkadapter which is deprecated

### ISSUE THREAD

**Issue by KennethBWSong** [OfficeDev/Microsoft-Teams-Samples#1039]
Code here: https://github.com/OfficeDev/Microsoft-Teams-Samples/blob/3a3049715a3710006304769b76bca00577bc102b/samples/bot-sso-adaptivecard/nodejs/index.js#L24

**Comment by Meghana-MSFT** [comment]
@KennethBWSong - Thank you for reporting this, we will check this and get the sample updated if necessary. Thank you.

**Comment by Meghana-MSFT** [comment]
https://github.com/OfficeDev/Microsoft-Teams-Samples/pull/1042/files has been raised for this issue.

**Comment by KennethBWSong** [comment]
Any updates?

**Comment by Meghana-MSFT** [comment]
PR has raised for this: https://github.com/OfficeDev/Microsoft-Teams-Samples/pull/1042/files. Changes will reflect once the PR is merged.

**Comment by KennethBWSong** [comment]
@Meghana-MSFT Seems the PR is merged, and I got following error when debug with Teams Toolkit:
![image](https://github.com/OfficeDev/Microsoft-Teams-Samples/assets/63089166/83b4d1f0-6ef8-4c88-a166-7a24b9cb3b77)
![image](https://github.com/OfficeDev/Microsoft-Teams-Samples/assets/63089166/cf7c4a2e-ad36-41cc-a6cc-77457c906445)
Do you have idea on what's the reason?

**Comment by KennethBWSong** [comment]
@Meghana-MSFT Also, in this sample to get SSO token: 
![image](https://github.com/OfficeDev/Microsoft-Teams-Samples/assets/63089166/855f2b27-c32d-436a-8a8b-9a3a13f1b003)
type of the card is "application/vnd.microsoft.card.loginRequest" which is different from [here](https://learn.microsoft.com/en-us/microsoftteams/platform/task-modules-and-cards/cards/universal-actions-for-adaptive-cards/sso-adaptive-cards-universal-action#add-code-to-handle-an-access-token). Would you please help to confirm which is the correct way?

**Comment by Meghana-MSFT** [comment]
We will check this and get back to you.

**Comment by Meghana-MSFT** [comment]
> @Meghana-MSFT Seems the PR is merged, and I got following error when debug with Teams Toolkit: ![image](https://private-user-images.githubusercontent.com/63089166/286793802-83b4d1f0-6ef8-4c88-a166-7a24b9cb3b77.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE5MjQ4ODYsIm5iZiI6MTcwMTkyNDU4NiwicGF0aCI6Ii82MzA4OTE2Ni8yODY3OTM4MDItODNiNGQxZjAtNmVmOC00Yzg4LWExNjYtN2EyNGI5Y2IzYjc3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMDclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjA3VDA0NDk0NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTEwOWZiYTFjMzdhZGZkMjA5YzhkNDk5YmI1ZDhhZTJmZDVmODUwMDA2ZmVmODM2MTBkNDk1Mjg5ZWZhYjcyZTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.e-8EmHAMjUiaLW4CdjCIFAQbC03c0C6ll1faTpLLRMs) ![image](https://private-user-images.githubusercontent.com/63089166/286793884-cf7c4a2e-ad36-41cc-a6cc-77457c906445.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE5MjQ4ODYsIm5iZiI6MTcwMTkyNDU4NiwicGF0aCI6Ii82MzA4OTE2Ni8yODY3OTM4ODQtY2Y3YzRhMmUtYWQzNi00MWNjLWE2Y2MtNzc0NTdjOTA2NDQ1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMDclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjA3VDA0NDk0NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ0N2MxN2I0MmQ1NTllYTI3ZGQ0OGQwZGRmZGFlZjNlYTVkMzVmY2Q3ZDBkYjM4ZGZjNDUwNzUyYzRiNTdmZGEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.5z3FjzKqRlho9X2_8gGoq4vEkYUTAUacv-ZSYLu2Z9A) Do you have idea on what's the reason?

This is a known issue, bug has been raised for this already - https://github.com/MicrosoftDocs/msteams-docs/issues/8584

**Comment by KennethBWSong** [comment]
@Meghana-MSFT Any updates on this?

**Comment by ChetanSharma-msft** [comment]
Hello @KennethBWSong - Sorry for delay in response.
We are actively checking the updates on this issue and let you know if any.

**Comment by simone-montemezzani-3AP** [comment]
Any update on this? The samples are really hard to follow in general, let alone when they contain wrong code.

**Comment by lilyydu** [comment]
Sample is going to be archived due to BF deprecation.
